tbdex/rfq/create.go

```diff
@@ -0,0 +1,220 @@
+package rfq
+
+import (
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/TBD54566975/tbdex-go/tbdex"
+	"github.com/tbd54566975/web5-go/crypto"
+	"go.jetpack.io/typeid"
+)
+
+// Create creates an [RFQ]
+//
+// # An RFQ is a resource created by a customer of the PFI to request a quote
+//
+// [RFQ]: https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#rfq-request-for-quote
+func Create(from, to, offeringID string, payin PayinMethodWithDetails, payout PayoutMethodWithDetails, opts ...CreateOption) (RFQ, error) {
```

Comment:
TIL if consecutive parameters have the same type we can omit the type from all but the last